src/main/webapp/user-page.html

```diff
@@ -23,6 +23,7 @@
     <link rel="stylesheet" href="/css/main.css">
     <link rel="stylesheet" href="/css/user-page.css">
     <script src="/js/user-page-loader.js"></script>
+    <script src="/js/navigation-loader.js"></script>
```

Comment:
After some investigation, I think that Elaine added this line because the user page was missing navigation links and not really because the ML Translation feature needed it. Personally, I include such cleanups only if I was already editing that line (if not, I would put it in a separate PR). No change needed in this PR, but something else to think about.